### PR TITLE
`-miss` arg for attacks

### PR DESCRIPTION
**Pull request by Croebh**
Probably wrong, but I feel like I'm learning at the very least?

### CODE REVIEW

File: cogs5e/funcs/sheetFuncs.py [hunk 1]
```diff
@@ -120,6 +121,9 @@ def sheet_attack(attack, args, embed=None):
                 out += "**To Hit**: Automatic hit!\n"
             if args.get('crit'):
                 itercrit = args.get('crit', 0)
+            if args.get('miss'):
```
Comment (mommothazaz123) [inline cogs5e/funcs/sheetFuncs.py]: This should probably be right under the check for hit, and be elif

**Comment by mommothazaz123** [comment]
(tagging #73)

**Comment by Croebh** [comment]
What would I need to change for init?

**Comment by mommothazaz123** [comment]
Actually nothing, apparently I didn't remember correctly.

**Pull request merged**